ddtrace/contrib/internal/langchain/patch.py

```diff
@@ -965,6 +965,166 @@ def traced_similarity_search(langchain, pin, func, instance, args, kwargs):
     return documents
 
 
+@with_traced_module
+def traced_base_tool_invoke(langchain, pin, func, instance, args, kwargs):
+    integration = langchain._datadog_integration
+    tool_input = get_argument_value(args, kwargs, 0, "input")
+    config = get_argument_value(args, kwargs, 1, "config") if len(args) >= 2 else None
```

Comment:
So from looking at langchain's [code](https://github.com/langchain-ai/langchain/blob/master/libs/core/langchain_core/tools/base.py#L390-L395) it looks like `config` is an optional keyword argument. This means there may be a case where `len(args)` is not 2 but it is present in `kwargs`, but this code will always return `None` in that case.
```suggestion
    config = get_argument_value(args, kwargs, 1, "config", optional=True)
```